packages/flutter/lib/src/gestures/events.dart

```diff
@@ -2241,6 +2290,7 @@ mixin _CopyPointerPanZoomEndEvent on PointerEvent {
       device: device ?? this.device,
       position: position ?? this.position,
       embedderId: embedderId ?? this.embedderId,
+      viewId: viewId ?? this.viewId,
```

Comment:
nit: Move this to the top of parameter list to keep order consistent.

Reply:
Done, also fixed for the other two instances where this went out of order.